### ISSUE TITLE
Possible Suspect animation implementation(s).

### ISSUE THREAD

**Issue by theengineertcr** [eezstreet/SWATEliteForce#172]
Can be used on higher difficulty and before a suspect will open fire or flee. More "fakes" to say.

Package:SWATMaleAnimation
Anim set:SwatHostage
Anim name 1:ComplyIndecision1
Anim name 2:ComplyIndecision3

**Comment by jose21crisis** [comment]
I looked them, and they are on SWATHostage for a reson, they are better for the hostages than the suspects........at least to me.

**Comment by theengineertcr** [comment]
How are these "hostages" holding guns then? The fake gun animation is there too(the one already implemented).

**Comment by eezstreet** [comment]
I agree with TCR, they're definitely meant for suspects judging by how they're posed in the first frame. 

I'm trying to think of how we can use them though. Maybe as alternatives to the existing one? But we'd need one for when they plan to comply, as well.

**Comment by theengineertcr** [comment]
True...I'll look into making these anims for complying first
